__tests__/data/intercom_output.json

```diff
@@ -304,8 +307,44 @@
     "files": {},
     "userId": "58b21c2d-f8d5-4410-a2d0-b268a26b7e33"
   },
+  {
+    "statusCode": 400,
+    "error": "Email or userId is mandatory"
+  },
```

Comment:
The test case output data order doesn't align with the inputs.

Reply:
That is another test case, remaining here from previous time. That does not include the new field. I have included a new test case later on. Should we remove this one then?

Reply:
Are all the TCs passing now?